interactors/src/main/java/com/thefuntasty/interactors/disposables/FlowableDisposablesOwner.kt

```diff
@@ -0,0 +1,173 @@
+package com.thefuntasty.interactors.disposables
+
+import com.thefuntasty.interactors.interactors.BaseFlowabler
+import com.thefuntasty.interactors.wrapWithGlobalOnErrorLogger
+import io.reactivex.Flowable
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.disposables.Disposable
+import io.reactivex.rxkotlin.plusAssign
+
+/**
+ * This interface gives your class ability to execute [BaseFlowabler] interactors
+ * and automatically add resulting disposables to one composite disposable. You
+ * may find handy to implement this interface in custom Presenters, ViewHolders etc.
+ *
+ * Consider using [DisposablesOwner] to support all of the basic RxJava 2 types.
+ *
+ * It is your responsibility to clear this composite disposable when all
+ * running tasks should be stopped.
+ */
+interface FlowableDisposablesOwner {
+
+    val disposables: CompositeDisposable
+
+    /**
+     * Executes the interactor and adds its disposable to
+     * shared, automatically disposed, composite disposable. In case some
+     * variant of [BaseFlowabler.execute] method has already been called
+     * on this instance of [BaseFlowabler], previous one is disposed,
+     * no matter what current state of internal Flowable is.
+     * Use [Flowable.executeStream] if you want to run one
+     * [BaseFlowabler] multiple times simultaneously.
+     *
+     * @param args Arguments used for initial interactor initialisation.
+     * @return disposable of internal [Flowable]. This disposable is disposed
+     * automatically. It might be used to dispose interactor when you need
+     * to dispose it in advance on your own.
+     */
+    fun <ARGS, T> BaseFlowabler<ARGS, T>.execute(args: ARGS): Disposable = execute(args, { })
+
+    /**
+     * Executes the interactor and adds its disposable to
+     * shared, automatically disposed, composite disposable. In case some
+     * variant of [BaseFlowabler.execute] method has already been called
+     * on this instance of [BaseFlowabler], previous one is disposed,
+     * no matter what current state of internal Flowable is.
+     * Use [Flowable.executeStream] if you want to run one
```

Comment:
Does it make a sense to have `executeStream` method when we have `disposePrevious` flag in the FlowablerConfig?

Reply:
Yes. It is useful if you want to do something like this:
```
interactor_A.create(Unit).flatMap {
    interactor_B.create(Unit)
}.executeStream()
```

Without `executeStream()` you have to do this:
```
disposables += interactor_A.create(Unit).flatMap {
    interactor_B.create(Unit)
}.subscribe()
```

Reply:
I see, but comments need to be updated because `Use [Maybe.executeStream] if you want to run one [BaseMayber] multiple times simultaneously.` this is incorrect. If you want to run one multiple times simultaneously then you should use `disposePrevious(false)` and if you want to execute stream then you should call `executeStream`

Reply:
Removed.

Reply:
```
In case some variant of [BaseCompletabler.execute] method has already been called on this instance of [BaseCompletabler], previous one is disposed, no matter what current state of internal Completable is.
```
I would add this
```
... This behavior can be disabled by passing false to [MaybeConfig.disposePrevious] method.
```
at the end of the previous sentence.

Reply:
Make sense. Done.